package.json

```diff
@@ -40,28 +40,22 @@
     "debug": "^2.1.1",
     "doctrine": "^1.1.0",
     "es6-map": "^0.1.3",
-    "escape-string-regexp": "^1.0.2",
-    "escope": "^3.4.0",
-    "espree": "^3.0.0",
+    "escope": "^3.3.0",
+    "espree": "3.0.0",
```

Comment:
here also, the version number are not correct for espree and escope vs the master.